app/controllers/api/databases.php

```diff
@@ -1640,8 +1640,55 @@ function updateAttribute(
         Event $queueForEvents
     ) {
         $key ??= $relatedCollectionId;
+        $twoWayKeyNull = is_null($twoWayKey);
```

Comment:
This is never used.

Reply:
Yep removed..
In the beginning, we wanted to auto-generate a key we realized that was not a good idea.